pyserini/2cr/atomic_html.template

```diff
@@ -0,0 +1,242 @@
+	background-repeat: no-repeat;
+	position: relative;
+	top: 3px;
+}
+
+.copy-code-button:focus {
+    /* Avoid an ugly focus outline on click in Chrome,
+       but darken the button for accessibility.
+       See https://stackoverflow.com/a/25298082/1481479 */
+    /* background-color: #E6E6E6; */
+	outline: 0;
+}
+
+pre[class*="prettyprint"] {
+	position: relative;
+	overflow: hidden;
+}
+    </style>
+</head>
+<body>
+
+    <!-- Background image -->
+    <div id="intro" class="bg-image vh-100 shadow-1-strong" style="max-height: 150px">
+      <div class="mask" style="
+            background: linear-gradient(
+              45deg,
+              rgba(29, 236, 197, 0.7),
+              rgba(91, 14, 214, 0.7) 100%
+            );
+          ">
+        <div class="container d-flex align-items-center justify-content-center text-center h-100"  style="max-height: 150px">
+          <div class="text-white">
+            <h1 class="mb-3">$title</h1>
+          </div>
+        </div>
+      </div>
+    </div>
+    <!-- Background image -->
+
+<div class="container my-4">
+
+<p>Key:</p>
+
+<ul>
+  <li>Validation: AToMiC validation dataset</li>
+  <li>Base: AToMiC base dataset</li>
+  <li>Large: AToMiC large dataset</li>
+</ul>
+
+<div class="table-responsive">
+  <table class="table">
+    <thead>
+      <tr>
+        <th class="headertop"></th>
+        <th class="headertop"></th>
```

Comment:
How about we add a description here to identify the query set?
E.g., val queries

Reply:
not too sure what you mean by this, does it correspond to any columns in the [data spreadsheet](https://docs.google.com/spreadsheets/d/1wSi_79Qx3GA1WAirwvoapiWJ4m2bPRM_rtUWRZ2qRIo/edit#gid=0)?

Reply:
I believe it would be beneficial to clarify that the results pertain to validation queries, as demonstrated in the [Miracl HTML template](https://github.com/castorini/pyserini/blob/2cab9cbf53139a821495f8bd46c5a45b14fea333/pyserini/2cr/miracl_html_table.template#L6). The data spreadsheet currently displays scores exclusively for the validation just for reference, but there are also test queries available.